dm/dm/master/server.go

```diff
@@ -1070,6 +1068,47 @@ func (s *Server) getStatusFromWorkers(ctx context.Context, sources []string, tas
 	}
 	wg.Wait()
 	s.fillUnsyncedStatus(workerResps)
+
+	getSubTaskStatusCountInResp := func(taskName string) int {
+		cnt := 0
+		for _, resp := range workerResps {
+			for _, status := range resp.SubTaskStatus {
+				if status.Name == taskName {
+					cnt += 1
+				}
+			}
+		}
+		return cnt
+	}
+	appendFakeResp := func(taskName, sourceName, msg string, cnt int) {
+		for i := 0; i < cnt; i++ {
+			setWorkerResp(&pb.QueryStatusResponse{
+				Result: false, Msg: msg, SubTaskStatus: []*pb.SubTaskStatus{{Name: taskName}},
+				SourceStatus: &pb.SourceStatus{Source: sourceName, Worker: "unknown-offline-worker"},
```

Comment:
Can we know and return the offline workers?

Reply:
we can get the workername from last bound 
but this may introduce new error and hard to deal with ( how should i handle the error from `GetLastSourceBounds`?)

Reply:
```suggestion
				SourceStatus: &pb.SourceStatus{Source: sourceName, Worker: "source not bound"},
```

this message also occur when no enough worker